ontotext-yasgui-web-component/src/plugins/yasr/charts-plugin.ts

```diff
@@ -0,0 +1,292 @@
+import {DownloadInfo, YasrPlugin} from '../../models/yasr-plugin';
+import {TranslationService} from '../../services/translation.service';
+import {SvgUtil} from '../../services/utils/svg-util';
+
+export class ChartsPlugin implements YasrPlugin {
+  // @ts-ignore
+  private yasr: Yasr;
+  // @ts-ignore
+  private translationService: TranslationService;
+  public static readonly PLUGIN_NAME = 'charts';
+  public label = "charts";
+  helpReference: string;
+  public priority = 7;
+
+  private chartEditor = null;
+  private wrapper = null;
+  private loaded = false;
+
+  // @ts-ignore
+  constructor(yasr: Yasr) {
+    if (yasr) {
+      this.yasr = yasr;
+      this.translationService = this.yasr.config.translationService;
+    }
+  }
+
+  canHandleResults(): boolean {
+    return !!this.yasr.results && this.yasr.results.getVariables() && this.yasr.results.getVariables().length > 0;
+  }
+
+  initialize(): Promise<void> {
+    console.log('1. init plugin', this.loaded);
+    return new Promise((resolve) => {
+      // One script tag loads all the required libraries!
+      const loader = document.createElement('script');
```

Comment:
You can use the "loadJavaScript" utility function of HtmlUtil

Reply:
used it